src/modules/python/pbs_hooks/PBS_cray_jacs.PY

```diff
@@ -324,16 +327,17 @@ def main():
     try:
         handler()
     except KeyboardInterrupt:
-        raise timeout_exc
+        raise timeout_exc('Handler timed out')
 
 
 if __name__ == '__builtin__':
     try:
         main()
-    except OfflineError:
-        vnode = HookHelper.get_event().vnode_list[pbs.get_local_nodename()]
-        vnode.state = pbs.ND_OFFLINE
-        vnode.comment = 'Node offlined due to Health Check Failure'
-        HookHelper.get_event().reject()
+    except OfflineError as e:
+        for vnode in pbs.event().vnode_list.keys():
+            vnode.state = pbs.ND_OFFLINE
+            vnode.comment = '%s: Node offlined due to JACS timeout: %s' % (
```

Comment:
Do we have to say "due to JACS timeout"?  Also, couldn't it be due to the hook alarm?

Reply:
I've changed it to print the message in the OfflineError exception.